utils/autotailor.8

```diff
@@ -57,16 +57,22 @@ Specify the rule to select. The rule ID can be either full, or just the suffix,
 Specify the rule to unselect. The argument works the same way as the --select argument.
 .RE
 .TP
-\fB-p NEW_PROFILE_ID, --new-profile-id NEW_PROFILE_ID\fR
+\fB-p TAILORED_PROFILE_ID, --tailored-profile-id TAILORED_PROFILE_ID\fR
 .RS
 Specify the ID of the tailored profile. The ID of the new profile can be either its full ID, or the suffix, in which case the 'xccdf_<id-namespace>_profile_' prefix will be prepended internally.
 If left out, the new ID will be obtained by appending '_customized' to the tailored profile ID.
 .RE
 .TP
+\fB--new-profile-id NEW_PROFILE_ID\fR
+.RS
+Synonym of --tailored-profile-id.
+.RE
+.TP
 \fB--json-tailoring JSON_TAILORING_FILE\fR
 .RS
 Import tailoring from a JSON file (https://github.com/ComplianceAsCode/schemas/tree/main/tailoring). This option makes BASE_PROFILE_ID positional argument optional.
 However, data passed in the command line options takes precedence over JSON contents, including the BASE_PROFILE_ID argument.
+When a JSON tailoring is accompanied with additional adjustments from the command-line options either a new profile would be added to the resulting XCCDF tailoring file or the profile with TAILORED_PROFILE_ID identifier, if present, will be updated accordingly.
```

Comment:
It doesn't seem to work the way it's documented here. For example:

```
utils/autotailor --id-namespace "com.example.www" --json-tailoring tests/utils/custom.json \
 --select R999 tests/utils/data_stream.xml.


Updating an existing profile with `--tailored-profile-id JSON_P1` seems to work.
```

should create a new profile but it doesn't create a new profile, the output is the same as if run without the `--select R999` option.

Reply:
Well, okay, the thing is that if you actually want a new profile you absolutely must provide base profile id. I can't make it a required parameter if --json-tailoring is present since it is ambiguous what the user actually wants (and depends on the content of the JSON file). I don't want to ruin backward compatibility, so that's what I came up with. If you have any suggestions on how to work around this, I'm all ears.

Reply:
My suggestion is to rephrase the doc text and help text. It needs to be clear that if the --json-tailoring option is set and other options are set at the same time, then the user needs to add `--tailored-profile-id`.

Reply:
It's not about `--tailored-profile-id` (it could be auto-generated) but about BASE_PROFILE positional argument. Which we can't deduce.

Reply:
but `--tailored-profile-id` works for me as well

Reply:
I've reworded the help message a bit.